caffe2/python/operator_test/conv_test.py

```diff
@@ -55,6 +70,9 @@ class TestConvolution(hu.HypothesisTestCase):
            engine=st.sampled_from(["", "EIGEN"]),
            shared_buffer=st.booleans(),
            use_bias=st.booleans(),
+           force_algo_fwd=_cudnn_convolution_algo_count("fwd"),
```

Comment:
this one doesn't test with "CUDNN" engine, so no need to change this one

Reply:
Ok.